src/main/build/version.h

```diff
@@ -16,7 +16,7 @@
  */
 
 #define FC_VERSION_MAJOR            2  // increment when a major release is made (big new feature, etc)
-#define FC_VERSION_MINOR            4  // increment when a minor release is made (small new feature, change etc)
+#define FC_VERSION_MINOR            3  // increment when a minor release is made (small new feature, change etc)
```

Comment:
Your change should not mess with INAV version. There are another PR just to bump the firmware version.

Reply:
@OlivierC-FR could you fix that maybe?

Reply:
done gentlemen !